### ISSUE TITLE
[Feature] Play of externally hosted videos with iframe

### ISSUE THREAD

**Issue by pcaglar** [stashapp/stash#3930]
**Is your feature request related to a problem? Please describe.**
We should be able to play videos hosted on Google photos or other cloud services.

**Describe the solution you'd like**
For example, we can play videos with iframe with lh3 links in google photos. Or we can play directly with the embed given by the site.

**Comment by HijackHornet** [comment]
If they are not hosted locally, how do you calculate their PHash ?
How do you detected dead links ? 
I'm not saying it's a bad Idea, just that it would require lots of adjustments to how the app work.

**Comment by pcaglar** [comment]
> If they are not hosted locally, how do you calculate their PHash ? How do you detected dead links ? I'm not saying it's a bad Idea, just that it would require lots of adjustments to how the app work.

You're right, you can't calculate phash. But I think an additional alternative source is always useful.

**Comment by estellaarrieta** [comment]
@pcaglar 
I too think that Google Photos support if too far-fetched.
But I can imagine a plugin that can open a file in Google Photos if it is uploaded there.
The idea is:
For every file in stash, generate a sha1 hash.
Make a plugin that adds a `Open in Google Photos` button to the scene page.
When button is clicked, plugin checks if the file is in Google Photos by hash, and if it is, opens its url.
Here is a tool that can do the file check. https://github.com/xob0t/gphotos_mobile_client
